README.md

```diff
@@ -47,6 +61,36 @@ To move a window, hold your `moveModifiers` down, then click and drag a window.
 
 To resize a window, hold your `resizeModifiers` down, then click and drag a window.
 
+### Fancy zones
```

Comment:
If the window isn't the size of the zone that I snap it to, I'm finding that the window _only_ resizes, but does not move.

Is this intentional, or a bug? I'd expect it to resize, then move.

https://user-images.githubusercontent.com/59429/144762310-479810bb-e606-4feb-b117-3b63d956ac7a.mov

You might have to do a resize, then set a timeout to fire the move X milliseconds later to avoid conflict.

Reply:
Whoops, I don't think I uploaded the correct video… I can reshoot if you need a visual description to go along with the text.

Reply:
Yeah, it is a bug, nice catch.

I am not able to reproduce it. When the click is released it should perform both resize and movement

```lua
function SkyRocket:handleCancel()
  return function()
    if not self.dragging then return end

    if self:isResizing() then
      self:resizeWindowToCanvas()
    elseif self:isMoving() then
      self:moveWindowToCanvas()
    elseif self:isFancy() then
      self:moveWindowToCanvas()
      self:resizeWindowToCanvas()
    end

    self.showcanvas = false
    self:stop()
  end
end
```

The behaviour that I am experiencing is as follows

https://user-images.githubusercontent.com/9949680/145725841-23b2a653-42c9-49e6-8223-eb565dbbf009.mov